build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -63,10 +63,10 @@ public override string ToString()
         // Full versions and package information.
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
-        public bool EnsureStableVersion => false;
-        public VerInfo LatestHostVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostPolicyVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        public bool EnsureStableVersion { get; set; }
+        public VerInfo LatestHostVersion => new VerInfo(Major, Minor, Patch, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
```

Comment:
> make sure we have the flexibility to have different versions if needed.

The flexibility exists in the change and we should define it when the need arises. In general, we would expect the versions of the packages to be always updated in sync with the M.N.App version as releases move on.